music21/abcFormat/__init__.py

```diff
@@ -1669,9 +1669,9 @@ def parse(self, forceKeySignature=None, forceDefaultQuarterLength=None):
 
         # Get the outer chord length modifier if present
         outer_lengthModifier = 1.0
-        if outerLengthModifierStr:
-            outer_lengthModifier = self.getQuarterLength(outerLengthModifierStr,
-                                                         forceDefaultQuarterLength=1.0)
+
+        outer_lengthModifier = self.getQuarterLength(outerLengthModifierStr,
+                                                    forceDefaultQuarterLength=1.0)
```

Comment:
misalignment of hanging indent (one space more)

Reply:
add this extra space and removed the previous line.